### ISSUE TITLE
Training locally

### ISSUE THREAD

**Issue by dani-lu** [katanaml/sparrow#20]
Hi Andrej, I tried to run your Fine Tuning colab locally but I had so much troubles with the Dataloaders and the multiprocessors, basically they weren't recognizing the donutDataset function defined in the notebook, so I had to define it outside and import it, but after this when Im training the model with GPU the training starts but the loss function never decreases. My question is, do you have a notebook or a script where you did a local training, or do you know how to fix this issue?, I would appreciate it a lot. 
Thank you for your attention.

**Comment by abaranovskis-redsamurai** [comment]
Hi. Yes, I do local training with this script (invoked from FastAPI endpoint) - https://github.com/katanaml/sparrow/blob/main/sparrow-ml/api/routers/donut_training.py
